pom.xml

```diff
@@ -76,6 +76,10 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.2</version>
+                <configuration>
```

Comment:
is this pom.xml change related to the property changes?

Reply:
No, it's not. I'll remove it.